inc/class-talk-settings-page.php

```diff
@@ -92,6 +104,26 @@ public function render_base_url_field() {
 			type="url"
 			value="<?php echo esc_url( get_option( 'coral_talk_base_url' ) ); ?>"
 		/>
+		<p class="description">The root url of the installed Talk application.</p>
+		<?php
+	}
+
+	/**
+	 * Prints input field for static url setting.
+	 *
+	 * @since 0.1.0
+	 */
+	public function render_static_url_field() {
+		?>
+		<input
+				style="width: 600px; height: 40px;"
+				name="coral_talk_static_url"
+				placeholder="https://cdn.talk-assets.com"
+				id="coral_talk_static_url"
+				type="url"
+				value="<?php echo esc_url( get_option( 'coral_talk_static_url' ) ); ?>"
+		/>
+		<p class="description">The root url where static Talk assets should be served from. This is used when static Talk assets are served from a CDN. (Default to the value of Talk Server Base URL)</p>
```

Comment:
It might be important to note that this is the same as the value for `process.env.TALK_STATIC_URI`.

> process.env.TALK_STATIC_URI
>
> Used to set the uri where the static assets should be served from. This is used when you want to upload the static assets through your build process to a service like Google Cloud Storage or Amazon S3 and you would then specify the CDN/Storage url. (Defaults to value of TALK_ROOT_URL)